samples/vertex-ffp/main.c

```diff
@@ -0,0 +1,190 @@
+/*
+ * This sample provides a very basic demonstration of 3D rendering on the Xbox,
+ * using pbkit. Based on the pbkit demo sources.
+ */
+#include <hal/video.h>
+#include <hal/xbox.h>
+#include <math.h>
+#include <pbkit/pbkit.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <strings.h>
+#include <xboxkrnl/xboxkrnl.h>
+#include <xboxrt/debug.h>
+
+#define pb_end(x) 0; pb_end(x)
```

Comment:
This is a hack to workaround an XGU issue (solved already?), where XGU uses new style pbkit code, but the XGU branch is based on a non-refactored pbkit.